dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_configurations_panel.tsx

```diff
@@ -378,12 +378,12 @@ export const DataConfigPanelItem = ({
               isTimeStampSelected
                 ? [...configList.span?.time_field]
                 : selectedObj?.label
-                ? [
+                  ? [
```

Comment:
There are several spacing changes .. make sure to check with prettier or eslint once. @harshada8989 please help here.

Reply:
Done !